### DISCUSSION TITLE
Is there a place for sharing "themes"?

### DISCUSSION THREAD

**Discussion by jahio** [starship/starship#2816] [Q&A] [answered]
Brand new user, looking to see if a certain prompt style ("theme") is already built by someone and shared somewhere. Is there some kind of gallery or website of starship prompts that people have shared? (If not, I'd suggest maybe making something like that to help new adopters.)

**Comment by matchai** [comment, marked as answer]
We currently have [a discussion](https://github.com/starship/starship/discussions/1107) and the [Presets page](https://starship.rs/presets/) where we collect themes.
There's definitely room for better discoverability here! 😊
